nep-Y.mediawiki

```diff
@@ -0,0 +1,328 @@
+<pre>
+  NEP: <to be assigned>
+  Title: NeoContract ABI
+  Author: Erik Zhang <erik@neo.org>, Roman Khimov <roman@nspcc.ru>
+  Type: Standard
+  Status: Draft
+  Created: 2022-12-08
+  Replaces: 14
+</pre>
+
+==Abstract==
+
+An Application Binary Interface (ABI) is the interface between two program modules, one of which is often a library and/or operating system and the other one is usually an application created by a regular programmer.
+
+This NEP describes the ABI standards for Neo smart contracts.
+
+==Motivation==
+
+Neo smart contract system is designed to be mutually invocable between contracts. To achieve this, we need a mechanism for exposing the interface of smart contracts. With NeoContract ABI, developers can easily create programs to invoke smart contracts or write clients that automatically access contract functionalities.
+
+This NEP is a compatible extension of the previous one, NEP-14. Some complex
```

Comment:
add link to https://github.com/neo-project/proposals/blob/master/nep-14.mediawiki?